src/main/scala/castalia/Receptionist.scala

```diff
@@ -20,24 +20,31 @@ object Receptionist {
 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/TimSoethout/Castalia/file/1938337595/issues/source?bid=2998213&fileBranchId=3098098#l26)